### PR TITLE
Adding netCDF4-python

### PR DESCRIPTION
**Pull request by ocefpaf**
Latest netCDF4-python compiled with Cython to get `filepath()` working.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Nothing is getting built... I will take a look later.

**Comment by pelson** [comment]
Also, you may need to disable xonsh for win64 (see https://github.com/conda-forge/staged-recipes/commit/22a0c2286169a98bf5448fef049d3cbe5e447fcf). (Feel free to do that as a separate PR.)

File: recipes/mingwpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    fn: mingwpy-0.1.0b3-cp34-none-win32.whl                                                           # [py34 and win32]
+    url: https://anaconda.org/carlkl/mingwpy/0.1.0b3/download/mingwpy-0.1.0b3-cp34-none-win_amd64.whl # [py34 and win64]
+    fn: mingwpy-0.1.0b3-cp34-none-win_amd64.whl                                                       # [py34 and win64]
+    url: https://anaconda.org/carlkl/mingwpy/0.1.0b3/download/mingwpy-0.1.0b3-cp27-none-win32.whl     # [py27 and win32]
+    fn: mingwpy-0.1.0b3-cp27-none-win32.whl                                                           # [py27 and win32]
+    url: https://anaconda.org/carlkl/mingwpy/0.1.0b3/download/mingwpy-0.1.0b3-cp27-none-win_amd64.whl # [py27 and win64]
+    fn: mingwpy-0.1.0b3-cp27-none-win_amd64.whl                                                       # [py27 and win64]
+
+build:
+    skip: True # [linux or osx]
```
Comment (pelson) [inline recipes/mingwpy/meta.yaml]: Perhaps we need to be more assertive: this is only for windows...

```
# [win]
```

Comment (ocefpaf) [inline recipes/mingwpy/meta.yaml]: I can change that in another PR.

Comment (pelson) [inline recipes/mingwpy/meta.yaml]: That was my thought. It wasn't important, but is nice for cleanliness. :+1:

**Pull request merged**